spec/controllers/calls_controller_spec.rb

```diff
@@ -2,11 +2,49 @@
 
 describe CallsController,  type: :controller do
 
-  def setup_active_call_double(target_legislators:[])
+  def setup_active_call_double(target_legislators:[], new_record:false)
     @active_call = double('active_call')
+    person = double('person')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.